pom.xml

```diff
@@ -201,7 +201,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>subversion</artifactId>
-            <version>2.7.1</version>
+            <version>2.13.0</version>
```

Comment:
Need to introduce a POM property so that the `tests` JAR version (line ~226) is also updated.

Reply:
Good catch, I missed that.

Reply:
Done